nixos/modules/services/networking/ssh/sshd.nix

```
@@ -438,6 +438,7 @@ in
                 "hmac-sha2-512-etm@openssh.com"
                 "hmac-sha2-256-etm@openssh.com"
                 "umac-128-etm@openssh.com"
+                "hmac-sha2-512"
```

Thank you for the PR and the detailed description, I have just one comment: Would it make sense to use the same entries as Mozilla proposes?

```suggestion
                "hmac-sha2-512"
                "hmac-sha2-256"
                "umac-128@openssh.com"
```

I also read the blog article mentioned in the description of this option and they recommend the same set of entries. That would align the default value more closely with what the documentation says.